src/js/state/Clusters/selectors.js

```diff
@@ -1,9 +1,15 @@
 /* @flow */
 
 import type {State} from "../types"
+import zealot from "../../services/zealot"
 
 export default {
   id: (id: string) => (state: State) => state.clusters[id],
+  getZealot: (id: string) => (state: State) => {
```

Comment:
Something like this already exists in the `Tab/selectors.js`

Reply:
Switched to use that one instead 👍, thanks!